6tempcontrol

```diff
@@ -222,6 +222,17 @@ do
 
     # Numeric check to avoid script breakage should nvidia-smi return error, also acts as backup watchdog
 
+    #Shutdown rig if SHUTDOWN_TEMP reached
+    if [[ $CURRENT_TEMP -gt $SHUTDOWN_TEMP ]]
```

Comment:
We need an option so users can chose if they want the HOT_SHUTDOWN or not
`if [[ $CURRENT_TEMP -gt $SHUTDOWN_TEMP ]] && [[ $HOT_SHUTDOWN == "YES" ]]`
Also in 1bash, before SHUTDOWN_TEMP